src/mappings/trades.ts

```diff
@@ -62,21 +76,13 @@ export function onTradeReversion(event: TradeReversionEvent): void {
   let orderId = toOrderId(event.params.owner, event.params.orderId)
   log.info('[onTradeReversion] New Trade Reversion for orderId {}', [orderId])
```

Comment:
Not exactly part of this PR, but is the `orderId` sufficient information to determine which trade was reverted? I would imagine we would also need to know some additional context like `userAddress` and possibly `batchId`...

Reply:
This is a Graph order entity id, which is `${owner}-${orderId}`. The naming here is because the Graph requires entities to have an "id" string field that uniquely identifies it (which `orderId` in the smart contract does not).

So that means that the user address is included here. You are correct that we additionally need to know the batch ID. A few lines down we calculate exactly that to find the trade being reverted :smile: